README.md

```diff
@@ -1,38 +1,126 @@
-   the network changes over time. There is also a strong set of metrics for
-   users who want to keep track of their own node and channels, or just explore
-   and create their own lightning data visualizations.
-
-3. Last but not least, `lndmon` uses [Grafana](https://grafana.com/) as its
+     tag, which enables lnd to export metrics about its gRPC performance and
+     usage.
+   - These metrics provide insights such as how many bytes lnd is transmitting
+     over gRPC, whether any calls are taking a long time to complete, and other
+     related statistics.
+   - `lnd` build notes:
+      * If you are using a release build of `lnd` or `litd`, this was enabled
+        during build and no action is required.
+      * If you are building from source, make sure to include the `monitoring`
+        tag (i.e. `make && make install tags=monitoring`)
+   - Default `lnd` config options:
+      * `prometheus.enable=false`
+         - Need to change to `true` to enable.
+      * `prometheus.listen=127.0.0.1:8989`
+         - You may want to change to `0.0.0.0:8989` to listen on all interfaces.
+      * `prometheus.perfhistograms=false`
+         - Enabling this uses more RAM and disk space.
+   - Note: This data can be obtained from `lnd` without `lndmon` being
+     installed.
+
+2. `lndmon`: while lnd provides some information about gRPC performance
+   statistics directly via its own prometheus exporter, `lndmon` by far does
+   the heavy lifting with regards to metrics.
+   - With `lndmon`'s data, you can track routing fees over time, track how the
+     channel graph evolves, and have a highly configurable "crystal ball" to
+     forecast and de-escalate potential issues as the network changes over time.
+   -  There is also a strong set of metrics for users who want to keep track of
+      their own node and channels, or just explore and create their own
+      lightning data visualizations.
+   - See [Options](#options) below for `lndmon` config options.
+   - Note: This data can be obtained without lnd being built with the
+     `monitoring` tag and without the `prometheus.enable` config option enabled,
+     but it does require `lnd` to be listening for gRPC connections.
+
+3. Prometheus is a time series database that scrapes data from `lnd` and
+  `lndmon` and stores it for later analysis.
+
+4. Last but not least, `lndmon` uses [Grafana](https://grafana.com/) as its
    primary dashboard to display all its collected metrics. Grafana is highly
    configurable and can create beautiful and detailed graphs organized by
    category (i.e., chain-related graphs, fee-related graphs, etc). Users have
    the option of making their Grafana dashboards remotely accessible over TLS
    with passwords to ensure their data is kept private.
 
+
+# Architectural Layout
+```
+                                             gRPC Server (TCP/10009)  ╔══════════════════════╗           
```

Comment:
Re this and the mono drawing above, you should give `mermaid` diagrams a shot: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams